doc/guides/west/build-flash-debug.rst

```diff
@@ -60,7 +60,7 @@ exactly the same name you would supply to CMake if you were to invoke it with:
 
 .. tip::
 
-   You can use the :ref:```west boards`` <west-boards>` command to list all
+   You can use the :ref:`west boards <west-boards>` command to list all
```

Comment:
this was fixed upstream as well, remove this commit

Reply:
Done